model/storage-private/src/main/java/org/keycloak/models/cache/CacheRealmProvider.java

```diff
@@ -39,5 +39,7 @@ public interface CacheRealmProvider extends RealmProvider, ClientProvider, Clien
     void registerRoleInvalidation(String id, String roleName, String roleContainerId);
 
     void registerGroupInvalidation(String id);
-    void registerInvalidation(String id);
```

Comment:
One of the reasons we added this method is to allow register invalidations on the realm cache for whatever entry we want. Therefore, avoid having methods like those below that are specific to organizations.

Can we keep the previous contract?

Reply:
Yes, I can add a "generic" event and keep the old contract :+1: